react/Summary.tsx

```diff
@@ -8,25 +8,40 @@ import SummaryContextProvider from './SummaryContext'
 const messages = defineMessages({
   label: {
     id: 'admin/editor.checkout-summary.label',
-    defaultMessage: '',
   },
   title: {
     id: 'store/checkout-summary.Summary',
-    defaultMessage: '',
   },
 })
 
 const CSS_HANDLES = ['summaryTitle', 'summaryContent'] as const
 
-const Summary: StorefrontFunctionComponent<StorefrontSummaryProps> = ({
+interface InsertCouponResult {
+  success: boolean
+  errorKey: string
+}
+
+interface SiteEditorSummaryProps {
+  title: string
+}
+
+export interface SummaryProps {
+  coupon?: string
+  insertCoupon?: (coupon: string) => Promise<InsertCouponResult>
+  loading?: boolean
+  totalizers: Totalizer[]
+  total: number
+}
+
+function Summary({
   children,
   loading,
   totalizers,
   total,
   coupon,
   insertCoupon,
   title,
-}) => {
+}: PropsWithChildren<SummaryProps & SiteEditorSummaryProps>) {
```

Comment:
Could you move this to a `type Props = PropsWithChildren<SummaryProps & SiteEditorSummaryProps>`?

Reply:
Done!